api_server/src/lib.rs

```diff
@@ -225,7 +225,10 @@ impl ApiServer {
             Ok(ParsedRequest::GetMMDS) => self.get_mmds(),
             Ok(ParsedRequest::PatchMMDS(value)) => self.patch_mmds(value),
             Ok(ParsedRequest::PutMMDS(value)) => self.put_mmds(value),
-            Err(e) => e.into(),
+            Err(e) => {
+                info!("{}", e);
```

Comment:
Would it be better if we log the errors with `error!`? I think it will make easier to identify logs which corresponds to errors, which will make also easier to describe errors, without any prefix that put a tag on the error message.

Reply:
done